spec/requests/api/generic_events_controller_create_spec.rb

```diff
@@ -69,6 +69,14 @@
               },
               'code' => 'blank',
             },
+            {
+              'title' => 'Unprocessable entity',
+              'detail' => 'From location id the location relationship you passed has an id that does not exist in our system. please use an existing from location',
+              'source' => {
+                'pointer' => '/data/attributes/from_location_id',
```

Comment:
So I've noticed relationship validations always point at the attributes section of the document when they have their separate section in the request as per the json:api spec. Happy to update this separately using magic and a bit of patience.